app/controllers/detail-viewers.ts

```diff
@@ -67,6 +67,9 @@ function getTimeExtensionSummaryRows(timeExtensionEvent: Collection<Application>
   request.push(addSummaryRow(i18n.pages.detailViewers.timeExtension.request.whatYouAskedFor, [ i18n.pages.detailViewers.timeExtension.request.wantMoreTime ]));
   request.push(addSummaryRow(i18n.pages.detailViewers.timeExtension.request.reason, [ data.details ]));
   request.push(addSummaryRow(i18n.pages.detailViewers.timeExtension.request.date, [ moment(data.date).format(dayMonthYearFormat) ]));
+  data.evidence.forEach(i => {
+    request.push(addSummaryRow(i18n.pages.detailViewers.timeExtension.request.evidence, [ i.name ]));
+  });
```

Comment:
you are adding a row for each evidence, instead of adding just one row with an array of evidences

Reply:
try something like
`request.push(addSummaryRow(i18n.pages.detailViewers.timeExtension.request.evidence, data.evidence.map(evidence => evidence.name) ));`